client/components/purchase-detail/README.md

```diff
@@ -33,3 +33,4 @@ export default React.createClass( {
 - *isPlaceholder* (boolean) – determines whether or not to render shimmering placeholders
 - *target* (string) – target passed as the `target` prop for the `Button`
 - *title* (string) – string used as the text of the heading
+- *requiredText* (boolean) – adds a notice to the top, determines the text in that notice
```

Comment:
I might be wrong but I think we display the same notice everywhere. If that's really a case we could just update this component to accept a boolean that will trigger the display of this notice. What do you think?